src/ElmSharp/ElmSharp/Size.cs

```diff
@@ -70,6 +70,11 @@ public override int GetHashCode()
                 return Width.GetHashCode() ^ (Height.GetHashCode() * 397);
             }
         }
+        
+        public override string ToString()
+        {
+            return string.Format("{{Width={0} Height={1}}}", Width, Height);
```

Comment:
The behavior of 'string.Format(string, object, object)' could vary based on the current user's locale settings. Replace this call in 'Size.ToString()' with a call to 'string.Format(IFormatProvider, string, params object[])'.